src/com/intellij/idea/plugin/hybris/properties/impl/PropertiesServiceImpl.kt

```diff
@@ -64,14 +64,14 @@ class PropertiesServiceImpl(val project: Project) : PropertiesService {
     override fun containsLanguage(language: String, supportedLanguages: Set<String>) = supportedLanguages
         .contains(language.lowercase())
 
-    override fun findAutoCompleteProperties(query: String): List<IProperty> = findAllProperties()
-        .filter { it.key != null && it.key!!.contains(query) || query.isBlank() }
 
-    override fun getProperty(key: String): IProperty? = findAllProperties()
-        .firstOrNull { it.key != null && it.key == key }
+    override fun findProperty(key: String): String? = findAllProperties().filter { it.key == key }.map { it.value }.firstOrNull()
```

Comment:
for readability perspective it would be beneficial to have stream operation on a new line:
```
xxx
.filter { .. }
.map { .. }
.firstOrNull()
```